exercises/concept/high-score/.meta/exemplar.rb

```diff
@@ -1,31 +1,31 @@
-module HighScore
+class Scoreboard
   INITIAL_SCORE = 0
 
-  def self.blank
-    {}
+  def initialize(initial_scores = {})
+    @scores = initial_scores
   end
 
-  def self.add_player(score_board, name, initial_score = INITIAL_SCORE)
-    score_board[name] = initial_score
-    score_board
+  def add_player(name, initial_score = INITIAL_SCORE)
+    @scores[name] = initial_score
   end
 
-  def self.remove_player(score_board, name)
-    score_board.delete(name)
-    score_board
+  def remove_player(name)
+    @scores.delete(name)
```

Comment:
```suggestion
    scores.delete(name)
```

Because at some point, the attribute reader should be defined:

```ruby
attr_reader :scores
```

This is preferred, because if we mistype, as a developer, the name of the instance variable, we get errors relating to `nil`, rather than a more helpful exception.

Reply:
I don't think we've not taught `attr_reader` at this stage in the track, so we can't use it here.

Reply:
(We can, of course, choose to teach it in this exercise or before this, but that's a decision that needs to be made in order for us to be allowed to use it here)

Reply:
I saw "exemplar", so was not aware that it would go to the student.

Reply:
Yeah. See the [spec for examplar files](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-exemplar-implementation). Exemplars are the solutions we want the student to aim for at this point in their learning.